packages/tinacms/src/utils/index.ts

```diff
@@ -15,6 +15,7 @@ import { Client, LocalClient } from '../client'
 import type { TinaIOConfig } from '../client'
 import * as yup from 'yup'
 
+export * from './client'
```

Comment:
We're not actually using any of the stuff you've added in the `util` folder are we?

Reply:
I will delete this import